servicex/expandable_progress.py

```diff
@@ -156,13 +161,15 @@ def add_task(self, param, start, total):
     def update(self, task_id, task_type, total=None, completed=None, **fields):
 
         if self.display_progress and self.overall_progress:
+            if task_type.endswith("Transform"):
```

Comment:
I didn't check the code out - but you always seem to `and` `display_progress` and `overall_progress` - which I don't totally get - because you set one or both in the `deliver` function.  With this wouldn't there only be one behavior?

Reply:
Not far below (line 202) we have an `and` of `display_progress` and `not overall_progress` :-) One could in principle wrap the whole thing in an `if self.display_progress:` block but I didn't really want to play much with the existing code beyond what absolutely needed changing.